Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/TestCertificateContainer.kt

```diff
@@ -0,0 +1,114 @@
+package de.rki.coronawarnapp.coronatest.type
+
+import de.rki.coronawarnapp.covidcertificate.test.TestCertificate
+import de.rki.coronawarnapp.covidcertificate.test.TestCertificateData
+import de.rki.coronawarnapp.covidcertificate.test.TestCertificateQRCodeExtractor
+import de.rki.coronawarnapp.util.encryption.rsa.RSAKey
+import de.rki.coronawarnapp.vaccination.core.CertificatePersonIdentifier
+import de.rki.coronawarnapp.vaccination.core.personIdentifier
+import de.rki.coronawarnapp.vaccination.core.qrcode.QrCodeString
+import de.rki.coronawarnapp.vaccination.core.server.valueset.VaccinationValueSet
+import de.rki.coronawarnapp.vaccination.core.server.valueset.getDisplayText
+import okio.ByteString
+import org.joda.time.Instant
+import org.joda.time.LocalDate
+import java.util.Locale
+
+abstract class TestCertificateContainer {
```

Comment:
can it be a `sealed ` class ? in Kotlin 1.5 sealed classes can be in different files

Reply:
Nice! I didn't know that. :+1: 
I tried it just now and it seems it will only work if the files are in the same package, and currently they are split by package `pcr` and `rapidantigen`. :frowning: